proto/cluster.proto

```diff
@@ -236,12 +239,15 @@ message WorkerGroupSpec {
   map<string, string> ray_start_params = 7;
   // Optional. The volumes mount to worker pods
   repeated Volume volumes = 8;
+  // Optional. ServiceAccount used by head pod
+  // Note that the service account has to be created prior to usage here
+  string serviceAccount = 9;  
```

Comment:
let's follow the naming conversions and use `service_account` here.

Reply:
Done